pom.xml

```diff
@@ -183,6 +192,11 @@
       </exclusions>
     </dependency>
 
+    <dependency>
+      <groupId>io.jenkins.plugins</groupId>
+      <artifactId>byte-buddy-api</artifactId>
+    </dependency>
```

Comment:
not until https://github.com/jenkinsci/byte-buddy-api-plugin/issues/64 is resolved and included in a bom

Reply:
commit 0582bf7